src/pages/index.js

```diff
@@ -51,8 +55,19 @@ export default () => {
         noBezel
       >
         <YouTubeEmbed
-          url="https://www.youtube.com/embed/2aLyGwaRufY"
-          title="Why Internet - Learn about the digital divide and how you can help"
+          url="https://www.youtube.com/watch?v=V1u2l_8C75k"
```

Comment:
Hey @ivanleeswe, sorry for not actually catching this earlier, but there's an issue with this line. You pasted the normal "watch" URL for the YouTube video, but you actually need the "embed" URL. You can get the embed URL from the "Share" button and the "embed" tab on YouTube itself, or, I think you can just take the old embed URL and just change the video ID.

Could you submit a new PR that fixes this and the other video? You'll know if it works if the embed button actually works.